Lab 5 #1

```diff
@@ -53,69 +53,62 @@ void setup() {
 
 void loop() {
 
-
-
   if (IMU.accelerationAvailable()) {
     IMU.readAcceleration(ax, ay, az);
 
-   
-{
-  
-  
-    
-    Serial.print(delta_t);
-  // Serial.print('\t');
-   // Serial.print(ay);
-   //Serial.print('\t');
-   // Serial.println(az);
-  }
-  
- delta_t = t_now - t_old;
+    t_now = millis();
```

Comment:
I think this was the line that was really causing problems.  All of the other changes are mostly formatting and data types.